JASP-Engine/JASP/R/common.R

```diff
@@ -501,7 +501,8 @@ isTryError <- function(obj){
 			
 		} else {
 		
-			stop(paste("bad call to .unv() : v is \"", v, "\"", sep=""))
+			#stop(paste("bad call to .unv() : v is \"", v, "\"", sep=""))
+		  vs[length(vs)+1] <- v
```

Comment:
What happens here? Is this for cases when it is a character like ":"?

Reply:
Actually: .unv intends to throw an input error but instead crashes the execution. It makes more sense to do no transformation if the condition is not satisfied. Also with the b64 and d64 functions it crashes if (some of) the variable names are not in base 64 encoding

Reply:
Seems sensible to me too. Just remove the commented out line.